src/storages/system/systemstorage_tables.go

```diff
@@ -56,8 +56,12 @@ func (storage *SystemTablesStorage) GetInputStream(session *sessions.Session, sc
 
 	// Block.
 	block := datablocks.NewDataBlock(cols)
-	if err := ctx.tablesFillFunc(dbName, block); err != nil {
-		return nil, err
+
+	dbNames := ctx.allDatabaseNames()
```

Comment:
Hello, 
SystemTables only return the tables which belongs to the current database(session.GetDatabases()).
allDatabaseNames used for?

Reply:
I think `system.tables` shall return all tables which behaviors the same as ClickHouse.

Reply:
```
SELECT
    database,
    name
FROM system.tables
GROUP BY
    database,
    name

┌─database────┬─name───────────────────────────┐
│ system      │ asynchronous_metrics           │
│ system      │ merges                         │
│ system      │ clusters                       │
│ system      │ table_engines                  │
│ system      │ parts                          │
│ test_slhi60 │ as_foo                         │
│ test_slhi60 │ tab_str                        │
```

Reply:
I see, Thanks.
But this needs more things to be changed:
1. fillTablesFunc should return all the database tables in databases/databases_callback.go, no need allDatabaseNames() here.
BTW, *Context.go only for parameter passing by design :D
2. https://github.com/vectorengine/vectorsql/blob/master/src/planners/planner_show_tables.go#L27 
query := "select * from system.tables order by name asc"
change to 
query := "select * from system.tables"

3. Rewritten the ShowTables Planner in:
https://github.com/vectorengine/vectorsql/blob/master/src/executors/executor_show_tables.go#L12
3.1 Add FilterPlanner and OrderByPlanner, the final query is: "select * from system.tables where database=[currentdatabase] order name asc"